motoman_driver/include/motoman_driver/joint_trajectory_streamer.h

```diff
@@ -143,6 +145,11 @@ class MotomanJointTrajectoryStreamer : public JointTrajectoryStreamer
   static bool VectorToJointData(const std::vector<double> &vec,
                                 industrial::joint_data::JointData &joints);
 
+  // variables for point streaming
+  double time_ptstreaming_last_point_;  // time at which the last point was received
+  double dt_ptstreaming_points_;        // elapsed time between two received points
+  static const double ptstreaming_timeout_ = 3.0; // seconds

```

Comment:
Should we make this a parameter that defaults to 3.0 seconds?

Reply:
We can but I guess it is already covered by "Force a transition to the IDLE state if we get a message with no-points" right?